spec/factories/test.rb

```diff
@@ -20,7 +20,43 @@
     end
 
     trait(:with_meta_data) do
-      meta_data { {namespace: {test_meta: 'data', test: 'test_data'}} }
+      meta_data do
+        {
+          'test-meta-data' => {
+            '@xmlns' => {
+              'namespace' => 'custom_namespace.org',
```

Comment:
Same as above:
- unintended indention
- why do we have the `@xmlns` multiple times

Reply:
I don't see any wrong indents here

Reply:
Ah, sorry -- my bad!